packages/reassure-measure/src/measure-function.tsx

```diff
@@ -0,0 +1,40 @@
+import { performance } from 'perf_hooks';
```

Comment:
It's not always available in the global namespaces, depends on the runtime (speaking from experience :P) - safer to explicitly import it from `'perf_hooks'` ;)